pipeline-scripts/buildlib.groovy

```diff
@@ -576,6 +586,20 @@ def mock_merge_driver(repo_dir, files) {
     }
 }
 
+/**
+ * Clones ose (with origin as 'upstream')
+ * Sets OSE_DIR to ose repo directory
+ * @return Returns the directory where ose is cloned
+ */
+def initialize_ose_4() {
```

Comment:
We should not need to do this anymore. If we add the openshift rpm to ocp-build-data this will all be automatically cloned by doozer.

Reply:
yes but we use it to populate the ose alias which I think is still a good idea for efficiency. or does doozer do only one ose checkout for multiple builds that use it? I think maybe you made it that smart but haven't checked lately :)

Of course not cloning the entire repo would improve efficiency too.

Reply:
Yeah... just pull the one file you need directly and use that. Then let doozer do the full clone and build.